crates/project/src/lsp_command/signature_help.rs

```diff
@@ -1,94 +1,82 @@
 use std::ops::Range;
 
-use gpui::{FontStyle, FontWeight, HighlightStyle};
+use gpui::{FontWeight, HighlightStyle};
 use rpc::proto::{self, documentation};
 
 #[derive(Debug)]
+pub struct SignatureHelps {
```

Comment:
We'll need a different name for this struct.

"Helps" doesn't work as a plural, as "Help" is both a countable and uncountable noun.